netlify/functions/getVideoThumbnail.ts

```diff
@@ -10,14 +10,34 @@ const getImage = async url => {
 export const createBlurredImage = async (thumbnailUrl: string) => {
   const image = await getImage(thumbnailUrl);
 
-  const buffer = await sharp(image)
+  const thumbnailSharpObject = await sharp(image)
     .raw()
     .ensureAlpha()
     .resize(12, 12, { fit: 'inside' })
-    .toFormat(sharp.format.png)
-    .toBuffer();
+    .toFormat(sharp.format.png);
 
-  return buffer.toString('base64');
+  const backgroundSharpObject = await sharp(image)
+    .raw()
+    .ensureAlpha()
+    .resize(1200, 1200, { fit: 'inside' })
```

Comment:
Does it need to be this big, given we're blurring it? Just wondering about how much this adds to page size on projects with many videos.